src/app/pages/create-wallet/create-wallet.tsx

```diff
@@ -1,7 +1,44 @@
-import { Component, h } from 'preact';
+import { h, Component } from 'preact';
+import { route } from 'preact-router';
+import Button from 'preact-material-components/Button';
+import LayoutGrid from 'preact-material-components/LayoutGrid';
+import 'preact-material-components/Card/style.css';
+import './create-wallet.scss';
+import TopBar from '../../components/top-bar/top-bar.container';
 
 export class CreateWalletPage extends Component {
-  public render() {
-    return <div>---- CreateWalletPage ----</div>;
+  public render(props, state) {
+    return (
+      <div className="create-wallet-page">
+        <TopBar />
```

Comment:
`TopBar` component is not needed it's included in layout component, you just have to do a rebase in order to work.